tests/pi_test.py

```diff
@@ -12,4 +12,4 @@ def test_pi_nested():
 
 
 def test_pi_arccos():
-    assert_equal("\\arccos{-1}", pi, symbolically=True)
+    assert_equal("\\arccos{-1}", acos(-1, evaluate=False))
```

Comment:
@builtbywill this test caught my attention. It's called "test_pi_arccos" because acos(-1) == pi. So why did we change the output to not be pi anymore? 

I can feel my brain melting trying to remember what these tests do.

Reply:
@lily514 hm true, but in that case then it wasn't really testing `process_sympy`, it was testing the comparison logic inside of `assert_equal`

Reply:
I dunno, just all the other tests in this file are specific to pi. If this isn't testing pi, can we move it to trig_test instead?

Reply:
Sure I can move it